server/src/v1/Title/Service/title.service.ts

```diff
@@ -73,34 +72,30 @@ export class TitleService {
         return serializerService.serializeResponse('title_list', result)
     }
 
-    async createTitle(openedBy: string, payload: CreateTitleDto, buffer: Buffer): Promise<ISerializeResponse> {
+    async createTitle(openedBy: string, payload: any, buffer: Buffer): Promise<ISerializeResponse> {
```

Comment:
Any particular reason the payload type changed to `any`?
- assumption is that we missed changing this back while developing/testing the frontend for this feature.

Looking at `titles.controller.ts`

```
async createTitle(@Headers('authorization') bearer: string, @Req() req) {
  //..
}
```

We should be passing the nestjs decorator `@Body CreateTitleDto` to ensrue the request conforms to the accepted params in the request for that controller action.